mentor_match_web/app/main/routes.py

```diff
@@ -37,12 +37,12 @@ def index():
 
 @main_bp.route("/cookies")
 def cookies():
-    return redirect("http://docs.mentoring.service.civilservice.lgbt/cookies/")
+    return redirect("http://mentormatching.online/cookies/")
```

Comment:
Should these be https?

Reply:
Good catch!
Fixed in commit [0265462](https://github.com/mentor-matching-online/mentor-match/pull/12/commits/026546244d8577f15bba71d0d18cf3d1f26caf4f)